src/rpc/util.cpp

```diff
@@ -315,6 +315,17 @@ std::string RPCExamples::ToDescriptionString() const
     return m_examples.empty() ? m_examples : "\nExamples:\n" + m_examples;
 }
 
+bool RPCHelpMan::IsValidNumArgs(size_t num_args) const
+{
+    size_t num_required_args = 0;
+    for (size_t n = m_args.size(); n > 0; --n) {
```

Comment:
nit, could change condition to `n > num_args`?

Reply:
No, that would be incorrect